pkg/kritis/metadata/containeranalysis/cache.go

```diff
@@ -45,6 +45,11 @@ func NewCache() (*Cache, error) {
 	}, nil
 }
 
+// Close closes client connections
```

Comment:
nit: typo

Reply:
Hi @ooq Thanks for the review. I can't see any typo here. Is there any grammar mistake in this comment. Please let me know.